src/main/kotlin/no/nav/syfo/infrastructure/cronjob/Cronjob.kt

```diff
@@ -1,7 +1,7 @@
 package no.nav.syfo.infrastructure.cronjob
 
 interface Cronjob {
-    suspend fun run()
+    suspend fun run(): CronjobResult
```

Comment:
Endrer ene og alene for å kunne hente ut resultatet i testen.
Noen forslag til bedre måter å løse det på?

Reply:
Endte opp med å droppe denne for `PubliserForhandsvarselCronjob` og heller teste `VarselService` i stedet (cronjobben gjør bare noe logging utover å kalle servicen)  - kan vurdere å gjøre tilsvarende for denne cronjobben?

Reply:
Jepp, kan gjøre det 👍🏼